forum_leaderboard/forum_leaderboard.py

```diff
@@ -0,0 +1,101 @@
+"""
+Forum Leaderboard XBlock
+
+Shows the top threads for a given discussion ID by vote.
+"""
+from django.template import Template, Context
+
+import pkg_resources
+
+from xblock.core import XBlock
+from xblock.fields import Scope, Integer, String
+from xblock.fragment import Fragment
+
+import lms.lib.comment_client as cc
```

Comment:
Upstream may not like this use of LMS code in an XBlock but for now it seems like the best way to go, since DiscussionService is not ready.